packages-next/fields/src/types/relationship/views/index.tsx

```diff
@@ -113,7 +113,7 @@ export const Field = ({
 
   if (value.kind === 'cards-view') {
     return (
-      <Stack as="fieldset" gap="medium">
+      <FieldContainer as="fieldset">
```

Comment:
the `legend` element expects to have a `fieldset` element as its immediate parent. 
the `Stack` component negates this via its inherent behaviour (wrapping each child in a pre-styled element)